sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -368,7 +370,7 @@ public partial class DataFeed
         public string Id { get { throw null; } }
         public Azure.AI.MetricsAdvisor.Models.DataFeedIngestionSettings IngestionSettings { get { throw null; } }
         public bool? IsAdministrator { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<string> MetricIds { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyDictionary<string, string> MetricIds { get { throw null; } }
```

Comment:
Should we rename this since it is no longer a List of Ids?

Reply:
Not worth updating it for beta 2 since we're close to code complete date, and this would required changes in all languages. Created an issue to track it: https://github.com/Azure/azure-sdk-for-net/issues/16632